code/game/objects/structures/dresser.dm

```diff
@@ -5,6 +5,8 @@
 	icon_state = "dresser"
 	density = 1
 	anchored = 1
+	burn_state = FLAMMABLE
+	burntime = 25
```

Comment:
I'd make them deconstruct-able as well as destructible via fire... Perhaps using a crowbar or a combination of a screwdriver and crowbar (loosen the fasteners and then pry apart the boards?) to dismantle them into planks without resorting to pulling out a flamethrower.